### ISSUE TITLE
No multiple concurrent logins in a VM configuration. (Goal: Azure setup)

### ISSUE THREAD

**Issue by flickfeeder** [stascorp/rdpwrap#216]
I'm connecting (or trying) a Windows 10 N Enterprise VM on a Windows Server 2016 host. All are up to date and the wrapper config / installation looks fine. But as soon as I try to connect another session I get asked to disconnect my current session, as if no wrapper would be active. The screenshot shows it all:

![image](https://cloud.githubusercontent.com/assets/17354388/23827824/3510956e-06bd-11e7-857d-ba8b19d9a1d4.png)

The last time (two weeks ago) I configured the same setup but on 'real' instead of virtual hardware and then all was fine.
My goals is to run a 'rdp wrapped' Windows 10 VM on Azure. Yes I could run session directly on the server instead of on the VM but my use case does not allow me to do so. I other words I really like to have it on a virtualized Windows 10 installation.
The virtualization *should* have no influence as far as my reasoning goes (Since I'm connecting to the VM's RDP and not the Server)... Any clues what's going on here?

**Comment by binarymaster** [comment]
Are you using the same user (which logged in locally) for testing RDP?

**Comment by flickfeeder** [comment]
No, I tried other useraccounts, I tried with rdpcheck, I tried with 'Single session per user' unchecked. All results in popping up the disconnection request like in the screenshot :(

**Comment by stamepicmorg** [comment]
are you using remoteFX at your guest-machine?

**Comment by flickfeeder** [comment]
Yes I do, is that the problem? I'm able to test later on today!

**Comment by stamepicmorg** [comment]
Yep, see this issue: https://github.com/stascorp/rdpwrap/issues/127

**Comment by flickfeeder** [comment]
Oops - I tested and you are completely right! Now I just wonder if the same behaviour occurred when I connected to the bare metal (non-VM) install of Windows 10... Contrary to what many believe, RemoteFX between two Windows 10 (since RDP8 I thought) is possible.
Some indications that RemoteFX is enabled (besides the performance enhancement) are:
- an event has been logged telling a RemoteFX connection has been initialized
- in the connection bar a connection info icon is visible: 
![image](https://cloud.githubusercontent.com/assets/17354388/23958446/70ac5558-09a2-11e7-98fc-3679993f49d0.png)
- resizing the screen is only possible to fixed resolutions (I assume the ones supported by the host GPU), opposed to *any* resolution when RemoteFX is not enabled.
Thank you very much @stamepicmorg and  @binarymaster for helping me out!
